### ISSUE TITLE
Ammonite does not properly backtick a value with `/*` in its name

### ISSUE THREAD

**Issue by fstoqnov** [lihaoyi/Ammonite#663]
After importing Media Range `*/*` any further commands do not get processed and the entire console needs to be restarted for ammonite to work again - I am running ammonite on Windows 10 in the default bash console
```scala
$ amm 
Loading... 
Welcome to the Ammonite Repl 1.0.0 (Scala 2.12.2 Java 1.8.0_131) 
If you like Ammonite, please support our development at www.patreon.com/lihaoyi 
@ import $ivy.`com.typesafe.akka::akka-http:10.0.9` 
import $ivy.$ 
@ import akka.http.scaladsl.model.MediaRanges.`image/*` 
import akka.http.scaladsl.model.MediaRanges.`image/*` 
@ 2 + 2 
res2: Int = 4 
@import akka.http.scaladsl.model.MediaRanges.`*/*` 
import akka.http.scaladsl.model.MediaRanges.`*/*`
@ 2 + 2 
cmd4.sc:164: unclosed comment  
 */*   
^ 
cmd4.sc:180:  '}' expected but eof found. 
}  
^ 
Compilation Failed
```

**Comment by bblfish** [comment]
I can confirm that the same problem occurs on OSX terminal.  

It looks like it cannot even be imported as `_`. 

```scala
$ amm
Loading...
Welcome to the Ammonite Repl 1.0.0
(Scala 2.12.2 Java 1.8.0_131)
If you like Ammonite, please support our development at www.patreon.com/lihaoyi
hjs-ammonite@ import $ivy.`com.typesafe.akka::akka-http:10.0.9`
import $ivy.$
hjs-ammonite@ import akka.http.scaladsl.model.MediaRanges._
import akka.http.scaladsl.model.MediaRanges._
hjs-ammonite@ 2+2
cmd2.sc:164: unclosed comment
  */*,
  ^
cmd2.sc:183: '}' expected but eof found.
}
 ^
Compilation Failed
```

**Comment by bblfish** [comment]
There is a workaround

```scala
import akka.http.scaladsl.model.MediaRanges
val star_star = akka.http.javadsl.model.MediaRanges.ALL.asInstanceOf[MediaRanges.PredefinedMediaRange]
akka.http.scaladsl.model.headers.Accept(star_star)
```

**Comment by lihaoyi** [comment]
Minimal-er repro
```scala
@ def `*/*` = 1

defined function */*

@ 1
cmd1.sc:61: unclosed comment
  */*
  ^
cmd1.sc:77: '}' expected but eof found.
}
 ^
Compilation Failed

```

It looks like our backtick-or-not logic needs to be extended to take into account the fact that an identifier that has only valid operator characters may need to be backticked if it contains `/*` anywhere within it

**Comment by lihaoyi** [comment]
The code we'd need to fix is here https://github.com/lihaoyi/Ammonite/blob/master/amm/util/src/main/scala/ammonite/util/Model.scala#L88

@fstoqnov @bblfish if you'd like to take a crack at sending a PR, this one should be relatively easy to fix

**Comment by lihaoyi** [comment]
Fixed in #668
